doc/quickstart/pgrouting_quickstart.rst

```diff
@@ -16,9 +17,10 @@ pgRouting is an extension that adds routing and other network analysis functiona
 This Quick Start describes how to enable pgrouting in a PostgreSQL database, load data with osm2pgrouting, and run a basic shortest path query with the sample test data.
 
 .. contents:: Contents
-
+   :local:
```

Comment:
What is the `:local:` keyword for? I tried to look in the HowTo Wiki but could find it.

Reply:
It omits the title of the document, so you just get the headings within it. Makes it look better.